pkg/ui/seeders.go

```diff
@@ -61,6 +63,13 @@ func (tui *TUI) SeedIncidentsUI() {
 	var incidentsData [][]string
 
 	tui.IncidentOpts.Statuses = []string{constants.StatusTriggered}
+
+	// Override incidents limit when viewing triggered incidents assigned to self
+	if tui.AssginedTo == tui.Username {
+		tui.IncidentOpts.Limit = 25
```

Comment:
Should this be set as const var instead of hard code?

Reply:
Right, I've refactored the variable to a constant